src/components/TitleBar/ViewTitleBar.js

```diff
@@ -126,12 +123,21 @@ const ViewTitleBar = props => {
                             <FilterSelector />
                             <span ref={buttonRef}>
                                 <Button
-                                    small={isSmallScreen(width)}
-                                    dataTest="more-button"
+                                    className={classes.moreButton}
```

Comment:
No biggie, but to reduce code repetition, parts of the button could be fetched from e.g. `const getMoreButton = className =>` or `const getMoreButtonContent = () =>`

Reply:
Done